generators/client/templates/angular/_package.json

```diff
@@ -112,16 +114,18 @@
     "node": ">=6.9.0"
   },
   "scripts": {
-    "lint": "tslint 'src/main/webapp/app/**/*.ts' --force",
-    "lint:fix": "tslint 'src/main/webapp/app/**/*.ts' --fix --force",
+    "lint": "tslint '<%= MAIN_SRC_DIR%>app/**/*.ts' --force",
+    "lint:fix": "tslint '<%= MAIN_SRC_DIR%>app/**/*.ts' --fix --force",
     "tsc": "tsc",
     "tsc:w": "tsc -w",
+    "ngc": "ngc -p tsconfig-aot.json",
+    "cleanup": "rimraf '<%= MAIN_SRC_DIR%>app/**/*.js' '<%= MAIN_SRC_DIR%>app/**/*.js.map' <%= BUILD_DIR %>",
```

Comment:
But havent we configured the AOT files to be generated in the target folder then why are these files being created here?